catalyst_voices_packages/catalyst_cardano/catalyst_cardano/test/wallet-automation/keys.txt

```diff
@@ -0,0 +1,19 @@
+# .txt file
+//WALLET1
+WALLET1_USERNAME=test123
+WALLET1_PASSWORD=test12345678@
+WALLET1_SEED_WORD_1=stomach
```

Comment:
One consideration, the approach of having each word in a separate var means that you need to know in advance the number of words when you read it or you need to look for WALLET1_SEED_WORD_{n} pattern.

I think many programming languages allow to split a string by a comma, etc.

To allow different seed lengths I would keep all the words as the following:

```
WALLET1_SEED_WORDS=stomach,horn,rail,afraid,...
```

and later in the code:
```
let seed = readArg("WALLET1_SEED_WORDS")
let words = seed.split(",")
```

What do you think?